README.md

```diff
@@ -1,48 +1,117 @@
+
+### GraphQL: 
+GraphQL intergration has a dependancy towards the GraphQL.Client, and also includes all additions which come from Fussilade and Polly as well. This package comes with a query builder which translates your common response models into a query.   
+
+## Download / Install <a name="downloadinstal"></a>
+The Xablu.WebApiClient is written following the multi-target library approach. Meaning you can simply add the Xablu.WebApiClient package through NuGet. Install the NuGet package into your shared .NET Standard project and ALL Client projects. 
+
+* NuGet: Xablu.WebApiClient
+* PM> Install-Package Xablu.WebApiClient
 * Namespace: using Xablu.WebApiClient
 
-## Build Status: 
+## Build Status: <a name="buildstatus"></a>
 [![Build status](https://ci.appveyor.com/api/projects/status/5ey0sq4fn01t9o56?svg=true
 )](https://ci.appveyor.com/project/Xablu/xablu-webapiclient)
 ![GitHub tag](https://img.shields.io/github/tag/Xablu/Xablu.WebApiClient.svg)
 [![NuGet](https://img.shields.io/nuget/v/Xablu.WebApiClient.svg?label=NuGet)](https://www.nuget.org/packages/Xablu.WebApiClient/)
 [![MyGet](https://img.shields.io/myget/xabluhq/v/Xablu.WebApiClient.svg)](https://www.myget.org/F/Xablu.WebApiClient/api/v2)
 
-## Usage
+## Key Features <a name="features"></a>
 
-Before using the Xablu.WebApiClient plugin (via `CrossRestApiClient.Current`) make sure you configure the Xablu.WebApiClient plugin with at least the base address of the API you want to connect to.
-Failing to do so will result in a `NotConfiguredException` when using the plugin.
+The WebApiClient contains new features with respect to the previous version. The list of key features is depicted below: 
 
-> Note: it is no longer needed to supply a platform specific `HttpMessageHandler`, this is now build in into the plugin. This means that the configuration (as shown below) can all be done in 
-> your shared code.
+REST Client:
+  * Based on Refit                                          ✔
+  * Implemented Retry and Timeout from Polly                ✔
+  * Implemented Fusillade’s Priorities                      ✔
 
-Configuration is done through the `CrossRestApiClient.Configure` method, like this:
+GraphQL Client: 
+  * Based on GraphQL.Client                                 ✔
+  * Implemented Retry and Timeout from Polly                ✔
+  * Implemented Fusillade’s Priorities                      ✔
+  * Implemented a Custom, Object-Oriented Query Builder     ✔
 
-```
-CrossRestApiClient.Configure((opt) => opt.ApiBaseAddress = "https://api.mybackend.com");
-```
+## Example <a name="examples"></a>
 
-Once you have configured the Xablu.WebApiClient plugin you can start using it through the static `CrossRestApiClient.Current` property, like this:
+Make sure to check out more examples in the Console Test App in the package. Down here is an example call for connecting with a Web API service through Refit:
```

Comment:
@Legendami shouldn't here be a link to the console app source code?

Reply:
Then the console app first has to be merged finally :/

Reply:
done